core/bin/zksync_tee_prover/src/tee_prover.rs

```diff
@@ -150,7 +150,7 @@ pub struct TeeProverConfig {
 impl Default for TeeProverConfig {
```

Comment:
I'm a bit confused. Why do we have two `TeeProverConfig` structures? [1](https://github.com/matter-labs/zksync-era/blob/52907eb3b62a5f4d6b09afc4c0649674b01342bf/core/bin/zksync_tee_prover/src/tee_prover.rs#L139), [2](https://github.com/matter-labs/zksync-era/blob/52907eb3b62a5f4d6b09afc4c0649674b01342bf/core/bin/zksync_tee_prover/src/config.rs#L10).

Why isn't it possible to override these values? The fact that you need to recompile/build/release a new version just to alter these values is a bit odd.

Wouldn't it make sense to make these variables a part of the "configurable" config?

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/2764/commits/37af465165dd80a13a4ff96e79f501a69e75c0d9). ✅